src/nl/surf/eduhub_rio_mapper/cli.clj

```diff
@@ -173,20 +173,23 @@
       :delete-and-mutate delete-and-mutate)))
 
 (defn parse-args-getter [[type id & [pagina]]]
-  {:pre [(rio.loader/valid-get-actions type)]}
-  (-> (when pagina {:pagina pagina})
-      (assoc (if (= type "opleidingsrelatiesBijOpleidingseenheid") ::rio/opleidingscode ::ooapi/id) id
-             ::rio/type type)))
+  (let [xml-response (str/starts-with? type "xml:")
+        type (if xml-response (subs type 4) type)]
+    (assert (rio.loader/valid-get-actions type))
+    (-> (when pagina {:pagina pagina})
+        (assoc (if (= type "opleidingsrelatiesBijOpleidingseenheid") ::rio/opleidingscode ::ooapi/id) id
+               :xml-response xml-response
```

Comment:
Name of this key sounds like it contains some kind of XML response, not it being some boolean requesting for XML instead of JSON(?).  Either change it into something like `xml-response?` or even better `response-type` which holds `:json` or `:xml`.  Note: I am not sure the alternative is JSON.